app/src/main/java/com/example/androidlearning/ContactAdaptor.kt

```diff
@@ -0,0 +1,83 @@
+package com.example.androidlearning
+
+import android.content.ContentProviderOperation
+import android.provider.ContactsContract
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import android.widget.Toast
+import androidx.recyclerview.widget.RecyclerView
+import com.example.androidlearning.databinding.ContactListBinding
+
+class ContactAdaptor(private val contactList: List<ContactModel>): RecyclerView.Adapter<ContactViewHolder>() {
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ContactViewHolder {
+        val layoutInflater = LayoutInflater.from(parent.context)
+        return ContactViewHolder(ContactListBinding.inflate(layoutInflater,parent,false))
+    }
+
+    override fun onBindViewHolder(holder: ContactViewHolder, position: Int) {
+        holder.setContact(contactList[position])
+        holder.updateContact(contactList[position])
+    }
```

Comment:
Could you please let me know difference between onCreateViewHolder() and onBindViewHolder() ?

Reply:
onCreateViewHolder() to create and initialize the view holder object
onBindViewHolder() to fill the data to view within view holder object